Source/CesiumRuntime/Public/CesiumMetadataPrimitive.h

```diff
@@ -101,7 +71,7 @@ class CESIUMRUNTIME_API UCesiumMetadataPrimitiveBlueprintLibrary
       BlueprintCallable,
       BlueprintPure,
       Category = "Cesium|Metadata|Primitive")
-  static const TArray<FCesiumFeatureIdAttribute>&
+  static const TArray<FCesiumFeatureIDAttribute>&
   GetFeatureIdAttributes(UPARAM(ref)
```

Comment:
Should I keep this and `GetFeatureIdTextures`? Maybe we'd need to pass `CesiumPrimitiveFeatures` into this to retrieve the original classes from the `FCesiumFeatureIDSet`s.